eng/common/pipelines/templates/steps/verify-changelog.yml

```diff
@@ -13,6 +13,10 @@ parameters:
   default: false
 
 steps:
+  - pwsh: | 
+      pip install packaging==20.4
```

Comment:
I assume this is only interesting for the Python part of this correct? If so why not add it to python specific part? In particular I assume it is for this line https://github.com/Azure/azure-sdk-for-python/blob/master/eng/scripts/Language-Settings.ps1#L14 as such why not do the pip install of the dependency in that same powershell function?

Reply:
Sounds good.